3rdparty/process_benchmark.cpp

```diff
@@ -0,0 +1,275 @@
+#include <iostream>
+#include <string>
+#include "mpi.h"
+#include <chrono>
+#include <string>
+#include <g3log/logworker.hpp>
+#include <g3log/g3log.hpp>
+#include <g3log/std2_make_unique.hpp>
+#include <g3sinks/LogRotate.h>
+#include "../src/Rifle.h"
+#include "../src/Vampire.h"
```

Comment:
please correct the includes. If it does not pick up the local ```Rifle.h``` and ```Vampire.h``` then you have to fix your path